tasks/main.yml

```diff
@@ -2,6 +2,15 @@
   when: mr_provisioner_do_provision
   delegate_to: localhost
 
+- name: Turn the netboot flag off with timeout
```

Comment:
This is a bad idea, a random timeout is difficult to get right when you have a varied set of HW all with different firmware booting times. Also, what happens when you turn it off before it has asked for the files? or between serving the kernel and the initrd? How do you plan to deal with failures?